plugins/techdocs-backend/api-report.md

```diff
@@ -41,6 +42,12 @@ export class DefaultTechDocsCollator implements DocumentCollator {
   readonly visibilityPermission: Permission;
 }
 
+// @public
+export interface DocsBuildStrategy {
+  // (undocumented)
+  shouldBuild(entity: Entity): Promise<boolean>;
```

Comment:
Sorry I didn't catch this before!

For future backward-compatibility reasons, it may be best to make this argument an object with key `entity`.  It's easy to imagine adding additional inputs for a build/no-build decision in the future, and making it an object makes it easier to do so in a non-breaking way.